generators/app/index.js

```diff
@@ -159,7 +160,8 @@ module.exports = class extends Generator {
                 clearCoverage: "shx rm -rf coverage",
                 karma: "run-s clearCoverage karma-ci",
                 lint: "eslint ./**/webapp/**/*.js && prettier --check ./**/webapp/**/*.{js,xml}",
-                "lint-fix": "eslint ./**/webapp/**/*.js --fix && prettier --write ./**/webapp/**/*.{js,xml} --no-error-on-unmatched-pattern"
+                "lint-fix":
```

Comment:
This line break is not necessary...

Reply:
This happens automatically due to the prettier config of the project. I could undo the line break, but the next time some one runs prettier on this file it will occur again.
Should I undo the line break?

Reply:
Grml... I know why I don't like prettier. It misses the fuzzy logic... 😁 

Then let's keep it as is and we can merge it...